clubs.html

```diff
@@ -43,7 +43,7 @@ <h1>Projects</h1>
 
 <div class="fadein-container">
   {% assign session = site.clubs.last %}
-  <div class="club recent fadein">
+  <div class="club recent fadein" >
```

Comment:
is there a reason for the extra space or just a typo?

Reply:
some shoddy merging